bot/res/haskell

```diff
@@ -1,3574 +0,0 @@
-ToRunElems
-ToRunStyle
-ToRunStyleD
-ToSp
-ToStart
-ToStr
-ToStrAttr
-ToString
-ToStyle
-ToStyles
-ToTable
-ToTblGrid
-ToTblLook
-ToTexinfo
-ToTextile
-ToXml
-ToYaml
-TP
-T.pack
-Tr
-TR
-Trace
-TrackChanges
-TrackLine
-Trademark
-TrailingNewlines
-Transform
-Traversable
-Traversable.mapM
-Treat
-Tree
-T.replicate
-Trim
-Triple
-True
-Try
-TS
-T.singleton
-T.stripEnd
-Tt
-T.Text
-T.toUpper
-T.unpack
-TupleSections
-TWiki
-TWikiWithWarnings
-Twips
-Two
-TwoLines
-TwoParens
-TWParser
-Txt2Tags
-Txt2TagsNoMacros
-Type
-Type.
-Typeable
-TypeOf
-Types
-TypesDoc
-TypesEntry
-Typeset
-TypesList
-TypeStr
-TypeSynonymInstances
-TypeTable
-TZ
-Unable
-Uncategorized
-Undefined
-Underline
-Underscores
-Unescapes
-Unicode
-UnicodeChar
-Uniform
-Unindented
-Unique
-UniqueId
-UNIX
-Unknown
-UnknownTagType
-UnlessSmart
-Unqualified
-UnsignedByte
-UnsignedLong
-UnsignedRational
-UnsignedShort
-UntilHeaderAboveLevel
-UNTITLED
-Untyped
-UnwantedBlocks
-Up
-UPC
-Update
-Upper
-UpperAlpha
-Uppercase
-UpperRoman
-Uri
-URI
-URI.
-URIReference
-URIs
-URIs.
-URIString
-Url
-URL
-URL.
-URLs
-URLString
-URN
-US
-USA
-Use
-Used
-UsePrefix
-User
-UserComment
-UserDataDir
-USERNAME
-Uses
-UsesEuro
-UseTags
-UTCTime
-UTF
-Utf8
-UTF8
-UTF8.fromString
-UTF8.fromStringLazy
-UTF8.hPutStrLn
-UTF8.putStrLn
-UTF8.readFile
-UTF8.toString
-UTF8.toStringLazy
-UTF8.writeFile
-Utilities
-Utility
-UUID
-Val
-ValAttr
-ValidChar
-ValLowercase
-Value
-Values
-ValueToInlines
-ValueToJSON
-ValueToString
-Var
-Variable
-Variables
-Variant
-Varieties
-VarListEntry
-VBlock
-VBScript
-Verb
-VERB
-Verbatim
-VerbatimBlock
-VerbatimChar
-VerbInNote
-Verbose
-Verilog
-Version
-VertAlign
-Vertical
-VHDL
-ViewL
-ViewPatterns
-ViewR
-Visible
-V.null
-VRML
-V.toList
-W3CDTF
-Wait
-Walkable
-Wang
-Warning
-Warnings
-WARRANTY
-Was
-We
-WebTeX
-Whether
-While
-While1
-WhileEnd
-WhitePoint
-Width
-WidthParameter
-Widths
-Will
-Windows
-Windows.
-WINDOWS
-With
-With3
-WithAttrs
-WithClass
-WithIds
-WithIndex
-WithLang
-WithM
-WithName
-WithNewline
-WithOrderedListMarker
-Without
-WITHOUT
-WithoutClasses
-WithParameters
-WithSpace
-WithWarnings
-Word
-Word16
-Word16be
-Word16le
-Word32
-Word32be
-Word32le
-Word64
-Word64be
-Word64le
-Word8
-WordChar
-Words
-Work
-Wrap
-Wrapper
-Wrapping
-Wraps
-WrapText
-WriteMode
-Writer
-WriterEnvironment
-WriterOptions
-WriterReader
-Writers
-WriterState
-WrongElem
-WS
-X.Attr
-XC
-XC.current
-XC.Cursor
-XC.fromContent
-XC.modifyContent
-XC.nextDF
-XC.root
-XC.toTree
-XhtmlFormatter
-XL
-Xml
-XML
-XML.
-XML.Attr
-Xml.defaultConfigPP
-XMLDoc
-XMLDocE
-Xml.elAttribs
-XML.Elem
-XML.Element
-Xml.ppcElement
-Xml.qPrefix
-XMLSchema
-XML_Schema_Fictionbook_2.1
-Xml.useShortEmptyTags
-XResolution
-XSLT
-XXX
-XZ
-Yaml
-YAML
-Yaml.Array
-Yaml.Bool
-Yaml.decodeEither
-YamlException
-YamlLines
-YamlMark
-Yaml.Null
-Yaml.Number
-Yaml.Object
-YamlParseException
-Yaml.String
-Yaml.Value
-Yaml.YamlMark
-YCbCrCoefficients
-YCbCrPositioning
-You
-YResolution
-YYYY
-Z0
-Zone
-ZonedTime
```

Comment:
> yorickpeterse's hangman was too difficult and very broad in scope, including things that might not have shipped with haskell.
To support my claim, I shall quote adaedra:
> 21:47:21 adaedra: How are we even supposed to guess things like A97C

Discovering new things is pretty much the entire point of the game. Always winning isn't.